### ISSUE TITLE
pytest wheel shouldn't contain py.test-2.7 entry point

### ISSUE THREAD

**Issue by remcohaszing** [pytest-dev/pytest#1487]
The [pytest wheel](https://pypi.python.org/packages/2.7/p/pytest/pytest-2.9.1-py2.py3-none-any.whl#md5=8864aa3ffe74cd3a0923ab1ee3d3ab29) is built with a hardcoded entry point named `py.test-2.7`.

As a result, this executable is always installed in the environment.

One would expect this would run `py.test` using Python 2.7, but this really just refers to whatever Python version was used to install it.

```sh
$ mktmpenv --python python3
Using base prefix '/usr'
New python executable in /home/remco/.cache/virtualenvwrapper/tmp-d8a0ac7580ca90b/bin/python3
Also creating executable in /home/remco/.cache/virtualenvwrapper/tmp-d8a0ac7580ca90b/bin/python
Installing setuptools, pip, wheel...done.
virtualenvwrapper.user_scripts creating /home/remco/.cache/virtualenvwrapper/tmp-d8a0ac7580ca90b/bin/predeactivate
virtualenvwrapper.user_scripts creating /home/remco/.cache/virtualenvwrapper/tmp-d8a0ac7580ca90b/bin/postdeactivate
virtualenvwrapper.user_scripts creating /home/remco/.cache/virtualenvwrapper/tmp-d8a0ac7580ca90b/bin/preactivate
virtualenvwrapper.user_scripts creating /home/remco/.cache/virtualenvwrapper/tmp-d8a0ac7580ca90b/bin/postactivate
virtualenvwrapper.user_scripts creating /home/remco/.cache/virtualenvwrapper/tmp-d8a0ac7580ca90b/bin/get_env_details
This is a temporary environment. It will be deleted when you run 'deactivate'.
(tmp-d8a0ac7580ca90b) ~/.cache/virtualenvwrapper/tmp-d8a0ac7580ca90b$ pip install pytest
Collecting pytest
  Using cached pytest-2.9.1-py2.py3-none-any.whl
Collecting py>=1.4.29 (from pytest)
  Using cached py-1.4.31-py2.py3-none-any.whl
Installing collected packages: py, pytest
Successfully installed py-1.4.31 pytest-2.9.1
(tmp-d8a0ac7580ca90b) ~/.cache/virtualenvwrapper/tmp-d8a0ac7580ca90b$ which py.test-2.7 
/home/remco/.cache/virtualenvwrapper/tmp-d8a0ac7580ca90b/bin/py.test-2.7
```

I think the simplest solution would be to simply remove this entry point, as I personally don't see why anyone would ever use such a postfixed executable anyway.

**Comment by davehunt** [comment]
We've discussed removing all `py.test*` entry points in #1632

**Comment by nicoddemus** [comment]
Fixed in #1662
